### PR TITLE
Add SPARK_DEFAULT_SERIES_NAME to internal.py

### PR DESCRIPTION
**Pull request by itholic**
When we use `"0"` as a default name for Series (In fact, almost all cases), I think It seems better to specify using specific constant.

added a constant `SPARK_DEFAULT_SERIES_NAME ` and addressed related files.
(I'm not pretty sure `internal.py` is the proper file to place this constant though)

### CODE REVIEW

File: databricks/koalas/internal.py [hunk 1]
```diff
@@ -63,6 +63,8 @@
 
 HIDDEN_COLUMNS = {NATURAL_ORDER_COLUMN_NAME}
 
+SERIES_DEFAULT_NAME = "0"
```
Comment (ueshin) [inline databricks/koalas/internal.py:66]: How about `SPARK_DEFAULT_SERIES_NAME`?

Comment (itholic) [reply databricks/koalas/internal.py:66]: @ueshin 

Hmm, It looks better in terms of maintaining consistency with `SPARK_DEFAULT_INDEX_NAME`.

Thanks!

**Comment by ueshin** [comment]
I'm working on fixing the build failure for `Conda (Python, Spark, pandas, PyArrow) (3.7, 2.4.6, 1.0.4, 0.15.1)` at #1576.

**Comment by ueshin** [comment]
@itholic Could you push an empty commit to rerun tests?

**Comment by itholic** [comment]
@ueshin Yup! I just reran them, thanks :)

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**